CHANGELOG.rst

```diff
@@ -7,6 +7,30 @@ The format is based on `Keep a Changelog`_,
 and this project adheres to `Semantic Versioning`_.
 
 
+Version 1.9.0 (2024-08-22)
+--------------------------
+
+* Added: pseudo-streaming support with ``audb.stream()``,
+  which returns the new ``audb.DatabaseIterator`` object.
```

Comment:
There are two relative clauses that start with which and have a comma. 
I was unsure about the comma and whether one should use *that* instead for one or the other.

This seems to refer to [restrictiveness](https://beaconpointservices.org/commas-with-relative-clauses-which-that-who/) of relative clauses.

Reply:
Nice, that you found a rule for that. Looking at [restrictiveness](https://beaconpointservices.org/commas-with-relative-clauses-which-that-who/), it seems to me that both cases are a nonrestrictive phrase, hence we should use a comma and "which".

Reply:
I find it always hard to decide the restrictiveness in a single case.  
I often avoid them and try to rephrase. In any case this conversation can be resolved now.